app/views/govuk_publishing_components/components/_layout_footer.html.erb

```diff
@@ -4,6 +4,7 @@
   meta ||= []
   navigation ||= []
   with_border ||= false
+  show_licence = show_licence.to_s.strip == 'false' ? false : true
```

Comment:
Might be easier to approach this using `hide_licence ||= false` - I think it might make the logic easier to follow. Then your change on line 99 would be `<% unless hide_licence %>` etc.

Reply:
That makes more sense! Have sorted 👍